py21cmsense/theory.py

```diff
@@ -0,0 +1,152 @@
+"""A module defining an interface for theoretical predictions, eg. power spectra.
+
+This is simply so that one can calculate sample variance values for a particular
+theoretical model, with a uniform interface.
+
+To register a class as a theory power spectrum, simply decorate it with ``theorypower``.
```

Comment:
I'm a little confused by this statement, I only see the word "theorypower" used here in the repo. Is there something I'm missing?

Reply:
Ah, remnant from earlier code. It's updated now to correctly say to subclass from `TheoryModel`.